pkg/e2e/e2e.go

```diff
@@ -121,12 +124,26 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 			log.Printf("Error while adding upgrade version property to cluster via OCM: %v", err)
 		}
 
-		err = clusterutil.WaitForClusterReady(cluster.ID(), nil)
+		clusterConfig, _, err := clusterutil.ClusterConfig(cluster.ID())
+		if err != nil {
+			log.Printf("Failed looking up cluster config for healthcheck: %v", err)
+		}
+		kubeClient, err := kubernetes.NewForConfig(clusterConfig)
+		if err != nil {
+			log.Printf("Error generating Kube Clientset: %v\n", err)
+		}
+		ctx, cancel := context.WithTimeout(context.Background(), time.Hour*2)
```

Comment:
Did you want to make this timeout configurable? It is in [c-am-o](https://github.com/openshift/configure-alertmanager-operator/blob/bb029f42f2209a8bde284c7c81c53827b543d801/pkg/readiness/cluster_ready.go#L59-L62) FWIW. (This could be a separate PR.)

Reply:
I thought about it, but I don't think we have a real use-case for configuring it. YAGNI?

Reply:
IIRC there's a zillion other config knobs around this, including the clean count, success sleep, and failure sleep. But I'm sure you're right -- are the other knobs ever used?

Reply:
@2uasimojo I've added a config knob for this and tested it.